app/views/categories/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= form_for @category do |f| %>
+  <div class="col-lg-6 category-add-form">
+    <div class="input-group input-group-lg">
+      <%= f.text_field :name, class: "form-control", placeholder: "Category name" %>
+      <span class="input-group-btn">
+        <%= f.button ("Create!"), class: "btn btn-success" %>
+      </span>
+    </div><!-- /input-group -->
+  </div><!-- /.col-lg-6 -->
```

Comment:
It's always best practice to go through your code and remove and unnecessary comments